### PR TITLE
pin sqla_1.4 from baselayer; fix queries

### PR DESCRIPTION
**Pull request by profjsb**

### CODE REVIEW

**Review by dannygoldstein — COMMENTED** [review]
we should use `safe_aliased` here:

https://github.com/skyportal/skyportal/blob/master/skyportal/models/group.py#L29

**Comment by profjsb** [comment]
> we should use `safe_aliased` here:
> 
> https://github.com/skyportal/skyportal/blob/master/skyportal/models/group.py#L29

Done 5071533

File: skyportal/handlers/api/photometry.py [hunk 1]
```diff
@@ -398,27 +397,6 @@ def get_values_table_and_condition(self, df):
...
-                for c in self._column_args:
-                    c._make_proxy(self)  # noqa
-
-            @property
-            def _from_objects(self):
-                return [self]
-
-        # https://github.com/sqlalchemy/sqlalchemy/wiki/PGValues
-        @compiles(_photometry_values)
         def _compile_photometry_values(element, compiler, asfrom=False, **kw):
```
Comment (mcoughlin) [inline skyportal/handlers/api/photometry.py:400]: Is this actually used anywhere now either?

**Review by mcoughlin — APPROVED** [review]
Just one question from the compile stuff, otherwise can merge that branch.

File: skyportal/handlers/api/photometry.py (hunk 1 above)
Comment (profjsb) [reply skyportal/handlers/api/photometry.py]: Good point. Removing.

**Comment by profjsb** [comment]
failure unrelated.

**Pull request merged**